boards/common/esp_common/Makefile.dep

```diff
@@ -0,0 +1,7 @@
+# enable bootloader reset over USB, requires CDC ACM to be used
+ifneq (,$(filter stdio_cdc_acm stdio_tinyusb_cdc_acm,$(USEMODULE)))
```

Comment:
With #19037, the feature was moved to `usbus_cdc_acm` and `tinyusb_class_cdc` to be independent on STDIO.
```suggestion
ifneq (,$(filter usbus_cdc_acm tinyusb_class_cdc,$(USEMODULE)))
```